project.clj

```diff
@@ -4,5 +4,5 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :repositories [["choco.repos" "http://www.emn.fr/z-info/choco-repo/mvn/repository/"]]
-  :dependencies [[org.clojure/clojure "1.5.1"]
+  :dependencies [[org.clojure/clojure "1.7.0-beta1"]
```

Comment:
Just to clarify... I always encourage trying out libraries with the latest release candidate of Clojure (to see if anything breaks) but I'd rather keep those dependency changes in our local directories than committing them.